item-detail.html

```diff
@@ -421,8 +421,134 @@
       </div>
     </div>
 
+    <!-- 埋め込みJSでは、兄弟要素を書いたときにエラー時の処理で兄弟が消えるケースがあるので、div でくくります。 Diffが見やすいようにインデントしていません-->
+    <!-- cms-content-typeには実際のコンテンツタイプのIDを指定します-->
+    <div>
+    <div class="p-product-body" cms-item cms-content-type="color-me-items" style="visibility: hidden;">
+      <div class="p-product-body-inner">
+        <{if $product.model != ""}>
+        <div class="p-product-body__model">
+          <{$product.model}>
+        </div>
+        <{/if}>
+
+        <div class="p-product-body__name">
+          <!-- Spearlyの埋め込みコードに置き換え -->
+          {%= color-me-items_title %}
+        </div>
+
+        <{* ソーシャル共有ボタン *}>
+        <ul class="p-product-body__share-buttons">
+          <li>
+            <{line_button}>
+          </li>
+          <li>
+            <{twitter_tweet}>
+          </li>
+          <li>
+            <{facebook_like layout="button_count" width=100}>
+          </li>
+        </ul>
+        <{* // ソーシャル共有ボタン *}>
+
+        <{if $review_use_flg}>
+        <div class="p-product-body__review-link">
+          <a href="#review">
+            <span><{$review_item_num}></span>
+            件のレビュー
+          </a>
+        </div>
+        <{/if}>
+
+        <{* 販売価格（ログイン時は会員価格） *}>
+        <{if $members_login_flg && $product.discount_flg}>
+        <div class="p-product-body__price p-product-body__price--discount">
+          <{if $option_price.max != $option_price.min}>
+          <{$option_price.min}> &#12316; <{$option_price.max}>
+          <{else}>
+          <{$product.sales}>
+          <{/if}>
+
+          <{$product.discount_rate}>OFF
+        </div>
+        <div class="p-product-body__regular-price">
+          <{$product.regular_price}>
+        </div>
+        <{else}>
+        <div class="p-product-body__price">
+          <{if $option_price.max != $option_price.min}>
+          <{$option_price.min}> &#12316; <{$option_price.max}>
+          <{else}>
+          <{$product.sales}>
+          <{/if}>
+
+        </div>
+        <{/if}>
+        <{* // 販売価格（ログイン時は会員価格） *}>
+
+        <{* 付与ポイント *}>
+        <{if $product.shoppoint > 0}>
+        <div class="p-product-body__point">
+          <span><{$product.shoppoint}></span>
+          ポイント還元
+        </div>
+        <{/if}>
+        <{* // 付与ポイント *}>
+
+        <{* 商品説明文 *}>
+        <{if $product.explain != ""}>
+        <div class="p-product-body__description">
+          <!-- Spearlyの埋め込みコードに置き換え -->
+          {%= color-me-items_content %}
```

Comment:
```suggestion
          {%= sample-ab-testing_description %}
```

このフィールドのタグは、 `{%= コンテンツタイプID_フィールドID %}`の形式となっております。
実際のデータに合わせると上記のような形になります 🙇